pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/kits/KitDisplayNameTest.java

```diff
@@ -3,40 +3,55 @@
 import java.util.List;
 
 import org.broadinstitute.dsm.DbTxnBaseTest;
```

Comment:
This class was changes a little bit because it was using juniper kits while it should have been pepper kits. Now that we have two functionalities for creating kits, I changed this to use the correct one (otherwise tests would not pass)